src/header/header.scss

```diff
@@ -25,118 +24,88 @@
-      display: flex;
-      align-items: center;
-      height: 100%;
-    }
+    .iui-header-logo {
+      > .iui-label {
+        font-size: $iui-font-size;
+      }
 
-    &,
-    > nav {
-      overflow: hidden;
-      box-sizing: content-box;
-      margin-bottom: -$iui-xs;
-      padding-bottom: $iui-xs;
+      .iui-header-button-icon {
+        margin: 0 $iui-l + $iui-xs;
+      }
     }
 
-    .iui-chevron {
-      @include iui-icons-small;
-      flex-shrink: 0;
+    .iui-header-button-icon {
+      width: $iui-icons-default;
+      height: $iui-icons-default;
       margin: 0 $iui-xs;
+      display: inline-flex;
     }
-  }
 
-  > .iui-center {
-    justify-content: center;
+    .iui-user-icon {
+      @include iui-user-icon-size($size: $iui-icons-large, $font-size: $iui-font-size-small);
+    }
   }
 
-  > .iui-right {
-    justify-content: flex-end;
+  @media (max-width: 768px) {
+    .iui-header-breadcrumbs-button {
+      .iui-header-button-icon {
+        display: none;
+      }
 
-    @media (prefers-reduced-motion: no-preference) {
-      .iui-user-icon {
-        &,
-        * {
-          transition: all $iui-speed-fast ease-out;
-        }
+      .iui-header-button-icon + .iui-button-label {
+        margin-left: 0;
       }
     }
   }
 
-  .iui-button {
-    height: 100%;
-    border-radius: 0;
-  }
-
-  .iui-divider {
-    width: 1px;
-    height: $iui-baseline * 2;
-    @include themed {
-      background-color: t(iui-color-background-4);
+  @media (max-width: 1024px) {
+    .iui-header-logo > .iui-label {
+      display: none;
     }
   }
+}
 
-  &.iui-slim {
-    height: round($iui-baseline * 2.5) + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
-
-    > .iui-left {
-      .iui-header-logo {
-        > .iui-label {
-          font-size: $iui-font-size;
-        }
+@mixin iui-page-header-zones {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
 
-        .iui-header-button-icon {
-          margin: 0 $iui-l + $iui-xs;
-        }
-      }
+@mixin iui-page-header-left {
+  @include iui-page-header-zones;
 
-      .iui-header-button-icon {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin: 0 $iui-xs;
-        display: inline-flex;
-      }
-    }
+  > nav {
+    display: flex;
+    align-items: center;
+    height: 100%;
+  }
 
-    > .iui-right {
-      .iui-user-icon {
-        @include iui-user-icon-size($size: $iui-icons-large, $font-size: $iui-font-size-small);
-      }
-    }
+  &,
+  > nav {
+    overflow: hidden;
+    box-sizing: content-box;
+    margin-bottom: -$iui-xs;
+    padding-bottom: $iui-xs;
   }
+}
 
-  @media (max-width: 768px) {
-    &,
-    &.iui-slim {
-      .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            @include iui-display-none;
-          }
+@mixin iui-page-header-center {
+  @include iui-page-header-zones;
+  justify-content: center;
```

Comment:
dont think its needed.. This would center inner content horizontally, which is not the point :?

Reply:
Removed it and visuals didn't change, removed.